spanner/row.go

```diff
@@ -50,28 +50,30 @@ import (
 //
 // Supported types and their corresponding Cloud Spanner column type(s) are:
 //
-//	*string(not NULL), *NullString - STRING
-//	*[]string, *[]NullString - STRING ARRAY
-//	*[]byte - BYTES
-//	*[][]byte - BYTES ARRAY
-//	*int64(not NULL), *NullInt64 - INT64
-//	*[]int64, *[]NullInt64 - INT64 ARRAY
-//	*bool(not NULL), *NullBool - BOOL
-//	*[]bool, *[]NullBool - BOOL ARRAY
-//	*float32(not NULL), *NullFloat32 - FLOAT32
-//	*[]float32, *[]NullFloat32 - FLOAT32 ARRAY
-//	*float64(not NULL), *NullFloat64 - FLOAT64
-//	*[]float64, *[]NullFloat64 - FLOAT64 ARRAY
-//	*big.Rat(not NULL), *NullNumeric - NUMERIC
-//	*[]big.Rat, *[]NullNumeric - NUMERIC ARRAY
-//	*time.Time(not NULL), *NullTime - TIMESTAMP
-//	*[]time.Time, *[]NullTime - TIMESTAMP ARRAY
-//	*Date(not NULL), *NullDate - DATE
-//	*[]civil.Date, *[]NullDate - DATE ARRAY
-//	*[]*some_go_struct, *[]NullRow - STRUCT ARRAY
-//	*NullJSON - JSON
-//	*[]NullJSON - JSON ARRAY
```

Comment:
nit: formatting looks off here

Reply:
Other lines have space followed by tab for indentation.
For these 2 lines, it was spaces.
Hence, it was rendering correctly in VSCode, but not in Github.